app/assets/stylesheets/govuk-overrides.scss

```diff
@@ -49,3 +49,11 @@
     }
   }
 }
+
+// ______________________________________________________
+
+// Filters
+
+.autocomplete__input {
```

Comment:
I made the assumption that is was the most appropriate place to store the override?